app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -134,19 +136,20 @@ class MEJSPlayer {
    * @param {Array} playlistItemT Array which contains playlist item clip start and end times.  This is sent in from playlist items plugin, when creating a new instance of the player.
    * @return {void}
    */
-  getNewStreamAjax(id, url, playlistItemsT) {
+  getNewStreamAjax(url, isEnded, playlistItemsT) {
```

Comment:
After looking though this commit, I wonder if `startPlaying` might be a better parameter name than `isEnded` or maybe that is conflating the actual `ended` case with starting playing once the player has be reinitialized?

Reply:
Yes, I agree :+1: